### PR TITLE
Add cases for Conv2DMMVec4Program and Conv2dNaiveProgram

### PR DESCRIPTION
**Pull request by axinging**
For Conv2DMMVec4Program, this case tests fitA and fitB are true.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5637)

### CODE REVIEW

**Comment by axinging** [comment]
@qjia7 @haoyunfeix @xhcao @gyagp PTAL

File: tfjs-backend-webgpu/src/kernels/conv2d_webgpu_test.ts [hunk 1]
```diff
@@ -243,4 +270,408 @@ describeWebGPU('conv2d vec4', () => {
              116, 53,  110, 142
            ]));
      });
+
+  it('conv2d x=[1,8,8,64] f=[3,3,3,64] s=[1,1] d=1 p=same Conv2DMMVec4Program fitA == true and fitB == true',
```
Comment (qjia7) [inline tfjs-backend-webgpu/src/kernels/conv2d_webgpu_test.ts:274]: `f=[3,3,64,64]`?

**Review by qjia7 — APPROVED** [review]
LGTM with one nit.

File: tfjs-backend-webgpu/src/kernels/conv2d_webgpu_test.ts (hunk 1 above)
Comment (axinging) [reply tfjs-backend-webgpu/src/kernels/conv2d_webgpu_test.ts]: done

**Pull request merged**